<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UTF8 Principle</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>381</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="qstringRadBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>QString Mode</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="qstringGrp">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>351</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPlainTextEdit" name="plainTextEdit_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>331</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QRadioButton" name="unicodeRadBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Unicode/UTF-8 Mode</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="unicodeGrp">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>351</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="uniBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Unicode</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>U+ </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="unicode32">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>221</width>
        <height>30</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>(0x) Hex Code Only</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="utf8Btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>UTF-8</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>UTF-8  </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="utf8_1">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>110</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="utf8_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>110</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="utf8_3">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>110</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="utf8_4">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>110</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>190</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>=&gt;</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>501</x>
      <y>13</y>
      <width>231</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QFontComboBox" name="fontComboBox">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>340</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>380</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Switch Font</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>390</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>390</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ms : </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
